src/App.js

```diff
@@ -30,12 +30,14 @@ function App() {
         </Switch>
       </header>
       <main className="App__main">
-        <p>Content example 1</p>
-        <p>Content example 2</p>
-        <p>Content example 3</p>
-        <p>Content example 4</p>
-        <p>Content example 5</p>
-        <p>Content example 6</p>
+        <Switch>
+          <Route path="/menu">
+            <Menu />
+          </Route>
+          <Route path="/order">
+            <Order />
+          </Route>
+        </Switch>
       </main>
       <footer className="App__footer">
         <Navigation />
```

Comment:
I was checking the app, and I saw the navigation still uses the default styles. Probably you have it planned for later

Reply:
Yep, I'm doing deep styling later :D